cmd/standalone/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM golang:1.7
+FROM scratch
 # expose agent UDP ports and query port
 EXPOSE 5775/udp 6831/udp 6832/udp 16686
```

Comment:
when we do baggage whitelisting, we'll be able to demonstrate it in hotrod and make use of the http endpoint